nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -1,22 +1,33 @@
 from warnings import warn
 
-import matplotlib as mpl
-import matplotlib.pyplot as plt
 import numpy as np
-from matplotlib.colors import LinearSegmentedColormap, Normalize
 
 from nilearn import DEFAULT_DIVERGING_CMAP
 from nilearn.plotting._utils import (
     get_cbar_ticks,
     get_colorbar_and_data_ranges,
+    save_figure_if_needed,
 )
+from nilearn.plotting.cm import mix_colormaps
 from nilearn.plotting.surface._backend import (
+    SurfaceBackend,
     _check_hemispheres,
     _check_surf_map,
     _check_views,
 )
 from nilearn.surface import load_surf_data
 
+try:
+    import matplotlib as mpl
+    import matplotlib.pyplot as plt
+    from matplotlib.cm import ScalarMappable
+    from matplotlib.colorbar import make_axes
+    from matplotlib.colors import LinearSegmentedColormap, Normalize
+except ImportError:
+    from nilearn.plotting._utils import engine_warning
+
+    engine_warning("matplotlib")
+
```

Comment:
Finally I plan to put all matplotlib imports here. The module will display warning if engine is not installed, but it will still be importable.